user_info.go

```diff
@@ -13,3 +13,16 @@ func (c *WorkwxApp) GetUser(userid string) (*UserInfo, error) {
 	obj := resp.intoUserInfo()
 	return &obj, nil
 }
+
+// GetUserByDept 获取部门成员详情
+func (c *WorkwxApp) GetUserByDept(id int64, fetchChild bool) ([]*UserInfo, error) {
```

Comment:
This API returns a list of users, so a name like `ListUsersByDeptID` might be more accurate and self-documenting.